doc/uml/Class_Diagram

```diff
@@ -0,0 +1,49 @@
+@startuml
+class Data{
+-Idenitifer
+-Content
+}
+
+class Tag{
+-Identifier
+-Sortable Ordinal
++rename()
+}
+
+class Database{
+-Tags[]
+-Data[]
+-Relationships[]
++createData()
++deleteData()
++createTag()
++deleteTag()
+}
+
+class Relationship{
+-Relationship(Data, Tag)
++returnTag()
++returnData()
++changeRelationship()
+}
+
+class Constraint{
+~comparaters()
```

Comment:
The PlantUML documentation indicates this is `package private` which I don't think is relevant. I get what you're trying to represent here I think, but I don't know how to do it myself; I just know this isn't it.